packages/messaging/package.json

```diff
@@ -7,10 +7,13 @@
   "module": "dist/esm/index.js",
   "scripts": {
     "dev": "gulp dev",
-    "test": "karma start --single-run && yarn type-check",
+    "test": "run-p test:karma type-check lint",
```

Comment:
I'm not seeing run-p being added anywhere as a devDependency.

Reply:
Also are logs going to be clean if one of these errors out?

Reply:
run-p is part of npm-run-all. There's a lot of run-p and run-s usage within this repo so I'm assuming it's going to be fine?